src/css/custom.css

```diff
@@ -8,47 +8,66 @@
 @import url('https://fonts.googleapis.com/css2?family=IBM+Plex+Mono:wght@400;500;600;700&display=swap');
 
 @font-face {
-  font-family: 'Atyp'; /* You can name it anything */
-  src: url('/fonts/AtypDisplay-Bold.woff2') format('woff2'),
-       url('/fonts/AtypDisplay-Medium.woff2') format('woff2'),
-       url('/fonts/AtypDisplay-Regular.woff2') format('woff2'),
-       url('/fonts/AtypDisplay-Semibold.woff2') format('woff2');
-  /* font-weight: normal;
-  font-style: normal; */
+  font-family: 'Atyp';
+  src: url('/fonts/AtypDisplay-Regular.woff2') format('woff2') font-weight 400;
+  src: url('/fonts/AtypDisplay-Medium.woff2') format('woff2') font-weight 500;
+  src: url('/fonts/AtypDisplay-Semibold.woff2') format('woff2') font-weight 600;
+  src: url('/fonts/AtypDisplay-Bold.woff2') format('woff2') font-weight 700;
 }
 
 :root {
   --ifm-font-family-base: 'IBM Plex Mono', sans-serif;  /* Set the default font to IBM Plex Sans */
   --ifm-color-primary: #000;
   --ifm-color-primary-dark: #000033;
-  --ifm-color-primary-darker: #001ac9;
   --ifm-color-primary-darkest: #0048f8;
   --ifm-color-primary-light: #0073ff;
   --ifm-color-primary-lighter: #98beff;
   --ifm-color-primary-lightest: #ffffff;
   --ifm-code-font-size: 95%;
   --docusaurus-highlighted-code-line-bg: rgba(0, 0, 0, 0.1);
-}
 
-h1, h2, h3, h4, h5, h6 {
-  font-family: 'Atyp', -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+  /* NEw COLORS */
+  --background-color: #fff;
+  --text-color: #202020;
```

Comment:
not sure if there was some discussion on this, but this color looks darker than the designs

Reply:
agreed. let me double check. maybe I copied the wrong value

Reply:
the color was right but the font weight `300` was not being imported. fixing